app/models/user.rb

```diff
@@ -54,25 +54,62 @@ def administrated_projects
     scope
   end
 
-  def self.to_csv
-    @users = User.order(:id)
+  def self.to_csv(
```

Comment:
should the csv_line and effective_project_role also be moved to the presenter given they aren't using the native model methods.  It'll keep the optimized logic outside of the model, that won't matter for other use cases.